.teamcity/_self/projects/WebApp.kt

```diff
@@ -200,60 +183,6 @@ object BuildDockerImage : BuildType({
 			"""
 		}
 
-		script {
-			name = "Post PR comment with link"
-			conditions {
-				doesNotEqual("teamcity.build.branch.is_default", "true")
-			}
-			scriptContent = """
-				#!/usr/bin/env bash
-
-				export GH_TOKEN="%matticbot_oauth_token%"
-				chmod +x ./bin/add-pr-comment.sh
-				./bin/add-pr-comment.sh "%teamcity.build.branch%" "calypso-live" <<- EOF || true
-				<details>
-					<summary>Calypso Live <a href="https://calypso.live?image=registry.a8c.com/calypso/app:build-%build.number%">(direct link)</a></summary>
```

Comment:
Note that these links are slightly different, and the TeamCity workflow would come in later to edit comments with these links including `app:build-######`. However, the `app:commit-#####`-formatted URLs used in the GitHub Actions workflow appear to work as expected, if a bit different that they show an interim "status" screen before redirecting to the built preview site.

Reply:
Doing some historical digging, I did find a reference at p4TIVU-9IM-p2#comment-11126 that the `commit-` prefixed tags were proposed as an ideal alternative to the `build-` prefixed tags.

Reply:
Fix proposed at https://github.com/Automattic/dserve/pull/228